01-starting-with-data.Rmd

```diff
@@ -209,6 +209,10 @@ The [slice](reference.html#slice) `1:4` means, "Start at index 1 and go to index
 
 The slice does not need to start at 1, e.g. the line below selects rows 5 through 10:
 
+> ## Tip {.callout} 
+> R indexes are 1-based, not 0-based. That means that first element has an index of 1 instead of 0.
```

Comment:
I agree this is confusing. This lesson is targeted at novice programmers. One nice thing about R is that we do not have to spend time discussing 0-indexing. Why would novices expect it to be any other way?

If we do want to make note of it, we should at least give them the context: explain that some other languages, e.g. Python, start indexing from zero.